languages/csharp/sensitivity/nullable.tmt

```diff
@@ -0,0 +1,52 @@
+~extensions start~
+~extensions import *~
+~extensions end~
+
+~main body start~
+
+~template nullification_operator_negative start~
+string tmp42 = null;
+~[VAR_string@1]~ = (~[VAR_string@1]~ + tmp42) ?? "";
+~template end~
+
+~template coalescing_operator_negative start~
+var tmp42 = "not null string" ?? ~[VAR_string@1]~;
+~[VAR_string@1]~ = tmp42;
+~template end~
+
+~template coalescing_operator_positive start~
+var tmp42 = null ?? ~[VAR_string@1]~;
+~[VAR_string@1]~ = tmp42;
+~template end~
+
+~template cast_nullable_positive start~
+try {
+    int? tmp42 = null;
+    ~[VAR_string@1]~ = ~[VAR_string@1]~ + $"{(int) tmp42}";
```

Comment:
The interpolated string will throw `InvalidOperationException` since `tmp` has `null` as value; maybe `tmp42` should be assigned an `int`?